src/pages/junghoon/Main/components/Aside/Aside.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import "./Aside.scss";
+
+class Aside extends Component {
+  render() {
+    return (
+      <div className="main-right">
+        <div className="profileSetRight">
+          <div className="dieterRamsRight">
+            <img
+              alt="Dieter Rams Profile"
+              src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS_pcQ9Q1D3JOFDrDeZfWB8hNPQLUBrLJv85pPyaF85njqOPgMQI97xDJITAqQIR-t6G9Q&usqp=CAU"
+            />
+          </div>
+          <div className="westaIdRightSet">
+            <p className="westaIdRight">dieter_rams</p>
+            <p>Braun-Dessau-Germany</p>
+          </div>
+        </div>
+        <div className="storyBox">
+          <div className="storyBoxNav">
+            <p className="storyBoxTitle">스토리</p>
+            <p className="storyBoxSeeAll">모두 보기</p>
+          </div>
+          <div className="profileSetStory">
+            <div className="bauhausLogoStory">
+              <img
+                alt="wassily kandinsky"
+                src="https://artpil.com/wp-content/uploads/2017/06/Wassily-Kandinsky-350bio.jpg"
+              />
+            </div>
+            <div className="westaIdStorySet">
+              <p className="westaIdStory">wassily_kandinsky</p>
+              <p className="westaStoryTime">10분전</p>
+            </div>
+          </div>
```

Comment:
컴포넌트 분리 후(위 리뷰 참조) 각각의 리스트들을 받아온 데이터를 사용해서 map을 통해 구성하면 좋겠습니다!
```suggestion
          this.props.data.map((ele, idx) => {
            return (
              <div className="profileSetStory" key={idx}>
                  <div className="bauhausLogoStory">
                    <img
                      alt={ele.alt}
                      src={ele.url}
                    />
                  </div>
                  <div className="westaIdStorySet">
                    <p className="westaIdStory">{ele.name}</p>
                    <p className="westaStoryTime">{ele.time}</p>
                  </div>
                </div>
            );
          })
```